app/views/like/_like.html.erb

```diff
@@ -94,18 +94,22 @@
         cache: false
       });
 
+      // opens model to login before you can like;
       $("#open-login-like").click(function() {
         $("input[name='hash_params']").val($(this).attr("data-hashparams"));
       });
 
+      // triggers liking an unliking
       <% if current_user %>
         $('#like-button-<%= node.id %>').on('click',<% if node.liked_by(current_user.uid) %>clickliked<% else %>clicknotliked<% end %>)
-      
-        if(window.location.href.includes("#like")) {
-          $('#like-button-<%= node.id %>').click();
-        }
+       
+       // note useful for comments on notes but maybe useful for something else. 
+        // if(window.location.href.includes("#like")) {
```

Comment:
It would be better to remove unused code.

Reply:
I don’t think code should be deleted before you know the reason it was put there in the first place. As noted in its comment I am still investigating that and this PR is not labeled for review. If you have input on the function of it that would be useful. Thank you

Reply:
You could look through the blame view for this file to see when it was added and when removed, and see if you can track it that way!

Reply:
It actually doesn't do anything helpful I looked into it. But instead of removing it I will comment it out. I think I will open a new issue after this for implementing likes on comments (currently not working) and fix it up there. Is that okay with everyone? So the uncommented code will just be there for a bit.